styles.css

```diff
@@ -10,6 +10,7 @@ html {
 
```

Comment:
- [ ] Make sure the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins). Please take a look at the below screenshots to find the differences and work on them 👇 
- Figma desktop popup 👇 
![Screenshot (216)](https://user-images.githubusercontent.com/67087664/156327250-d2b2c69e-c2b7-43a0-a0bc-7d43c131d741.png)
- Your desktop popup 👇 
![Screenshot (212)](https://user-images.githubusercontent.com/67087664/156327457-46fa97c6-498b-4040-9608-1a374a70e88c.png)
- Figma mobile popup 👇 
![Screenshot (214)](https://user-images.githubusercontent.com/67087664/156327524-a7a09990-e488-45cd-b423-58583d9973ea.png)
- Your mobile popup 👇 
![Screenshot (213)](https://user-images.githubusercontent.com/67087664/156327584-cd1d862f-e774-4b31-91f9-56e96fec7e80.png)